### PR TITLE
Small dev documentation fix

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: docs/developer_guide.md [hunk 1]
```diff
@@ -222,8 +222,8 @@ the same region.
 
 ### Adding new charts
 
-- Update server/chart_config.json with the new chart.
-  - ```json
+1. Update server/chart_config.json with the new chart.
```
Comment (shifucun) [inline docs/developer_guide.md:225]: server/chart_config/*.json

File: docs/developer_guide.md [hunk 2]
```diff
@@ -239,32 +239,30 @@ the same region.
         }
       }
     ```
-  - If new stat vars are introduced, also update these files:
-    - static/js/i18n/strings/en/stat*var*[labels|titles].json.
-  - If adding new units, also add translations to ___
 
-- Update related files.
+1. Update related files.
+   - If adding a new category, create a new config file in [server/chart_config](../server/chart_config) and:
```
Comment (shifucun) [inline docs/developer_guide.md:244]: and add the new category to:

File: docs/developer_guide.md [hunk 3]
```diff
@@ -239,32 +239,30 @@ the same region.
...
+     - [i18n.tsx](../static/js/i18n/i18n.tsx)
+     - [en/units.json](static/js/i18n/strings/en/units.json) (with display names and labels for the unit)
 
-  - If a new unit is required, update:
-    - [i18n.tsx](../static/js/i18n/i18n.tsx) as well as
-    - [en/units.json](static/js/i18n/strings/en/units.json) with display names and labels for the unit.
+   Note: Please add very detailed descriptions to guide our translators. See localization.md for more details.
 
-  Note: Please add very detailed descriptions to guide our translators. See localization.md for more details.
+1. Run these commands:
```
Comment (shifucun) [inline docs/developer_guide.md:257]: maybe emphasize this is required for every change above

**Review by shifucun — APPROVED** [review]
Thanks for the fix!

File: docs/developer_guide.md [hunk 4]
```diff
@@ -239,32 +239,30 @@ the same region.
...
+   ```bash
+   ./scripts/extract_messages.sh
+   ./scripts/compiled_messages.sh
```
Comment (shifucun) [inline docs/developer_guide.md:260]: oh, this should be compile_messages

File: docs/developer_guide.md (hunk 3 above)
Comment (beets) [reply docs/developer_guide.md:258]: good point, updated

File: docs/developer_guide.md (hunk 4 above)
Comment (beets) [reply docs/developer_guide.md:261]: thanks for catching that

**Review by beets — COMMENTED** [review]
thanks for the review

File: docs/developer_guide.md (hunk 2 above)
Comment (beets) [reply docs/developer_guide.md]: thanks, added

File: docs/developer_guide.md (hunk 1 above)
Comment (beets) [reply docs/developer_guide.md]: added

**Pull request merged**